python/services/evaluationservice/dmod/evaluationservice/evaluation_service/templates/evaluation_service/ready_evaluation_async.html

```diff
@@ -0,0 +1,293 @@
+    }
+
+    let messageConfig = {
+        mode: "javascript",
+        json: true,
+        readOnly: true,
+        indentUnit: 4,
+        viewportMargin: Infinity,
+    };
+
+    let codeViews = [
+        {
+            "name": "editor",
+            "tab": "edit-div",
+            "container": "editor-content",
+            "view": null,
+            "config": {
+                mode: "javascript",
+                json: true,
+                indentUnit: 4,
+                lineNumbers: true,
+                allowDropFileTypes: ['application/json'],
+                viewportMargin: Infinity,
+                matchBrackets: true,
+                autoCloseBrackets: true,
+                foldGutter: true,
+                gutters: ["CodeMirror-linenumbers", "CodeMirror-foldgutter"]
+            },
+            "textarea": "#instructions"
+        },
+        {
+            "name": "messages",
+            "tab": "message-div",
+            "container": "message-area",
+            "view": null,
+            "config": {
+                mode: "javascript",
+                json: true,
+                readOnly: true,
+                indentUnit: 4,
+                viewportMargin: Infinity,
+            },
+            "textarea": "#messages"
+        }
+    ];
+
+    const LAUNCH_URL = "{{ launch_url }}";
+
+    let digest = {
+        digest: []
+    };
+</script>
+{% endblock scripts %}
+{% block style %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.6/addon/fold/foldgutter.min.css"
+          crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" type="text/css" href="{% static 'evaluation_service/css/ready_evaluation_async.css' %}">
+{% endblock style %}
+{% block bannercontent %}
+    <h1>Setup</h1>
+{% endblock bannercontent %}
+{% block content %}
+    <div id="content" class="pane">
+        <div id="launch-warning">
+            <p>
+                This page is for testing the evaluation service - use in production should be initiated directly via the
+                API, not here
+            </p>
+        </div>
+        <div id="general-error-box" class="error-box">
+            <p id="error-message"></p>
+            <img id="acknowledge-error-icon" src="{% static 'evaluation_service/img/error.png' %}"
+                 alt="I am aware of this error" onclick="updateError(null)">
+        </div>
+        <div id="tabs">
+            <button id="instruction-tab-button" class="tablinks" onclick="switchTabs(event, 'edit-div')">Instructions</button>
+            <button id="messages-tab-button" class="tablinks" onclick="switchTabs(event, 'message-div')">Messages</button>
+        </div>
+        <div id="edit-div" class="tab">
+            <div id="editor-header" class="tab-header">
+                <label for="evaluation_id">Evaluation ID:</label>
+                <input id="evaluation_id" name="evaluation_id" type="text" maxlength="100"
```

Comment:
For the purposes of this PR, I think this (inputs / form) is perfectly fine and what we need. However, in the future, we might consider using [`django` forms](https://docs.djangoproject.com/en/4.1/topics/forms/#building-a-form-in-django) in the future.

Reply:
_Luckily_, this view shouldn't be used that often (if ever). See `$("#launch-warning")` as to why. Also, I seem to really hate django form rendering for whatever reason. If that's what you've been using for some of the new pages in the main GUI this might be a good place for me to practice the new style.

Reply:
Thanks for the call out, ill give that a read. Yeah, to be fair, django forms are kind of wonky if you want to add js validation or do anything mildly complicated. I think they are only usable if you are cool styling them with `flexbox` or bet yet `grid`. I am using them in the [dataset manager view](https://github.com/NOAA-OWP/DMOD/blob/085dab4c534000b22c5c2924ba0df6c2b89528c2/python/gui/MaaS/cbv/DatasetManagementForms.py) for the at the moment.